hack/build.sh

```diff
@@ -50,7 +50,11 @@ function squash {
 dirs=${VERSION:-$VERSIONS}
 
 for dir in ${dirs}; do
-  IMAGE_NAME="${BASE_IMAGE_NAME}-${dir//./}-${OS}"
+  if [ "$dir" == "2.6" ]; then
```

Comment:
this will make sharing `build.sh` between multiple repos more complex. The refactor I did few weeks(?) ago attempted to unify the build.sh and Makefile so we don't have to manually patch each one in every repo when we do changes.